### PR TITLE
BDR-583 Additional properties in the metadata schema for templates

### PR DESCRIPTION
**Pull request by kehh**
* This creates necessary values in the schema so that we can place them in the frontend.
* Does not include refactoring of template id

### CODE REVIEW

File: abis_mapping/templates/dwc_mvp/metadata.json [hunk 1]
```diff
@@ -1,7 +1,11 @@
 {
     "id": "dwc_mvp.csv",
-    "description": "A template to translate some Darwin Core fields",
+    "label": "Occurrence Data Template",
```
Comment (kehh) [inline abis_mapping/templates/dwc_mvp/metadata.json:3]: I know the label doesn't match the id, but it'll help Jason on the way.

**Comment by HaydenGaia** [comment]
Looks good to me, merging :+1:

**Comment by HaydenGaia** [comment]
@kehh Actually, I don't see a merge button. Do I have to be added to this repo?

**Comment by kehh** [comment]
> @kehh Actually, I don't see a merge button. Do I have to be added to this repo?

Do you have a button now?

**Comment by HaydenGaia** [comment]
> > @kehh Actually, I don't see a merge button. Do I have to be added to this repo?
> 
> Do you have a button now?

Yes, thanks!

**Pull request merged**